packages/@react-spectrum/combobox/docs/ComboBox.mdx

```diff
@@ -1012,7 +1012,7 @@ it('ComboBox can select an option via keyboard', async function () {
   // Render your test component/app and initialize the combobox tester
   let {getByTestId} = render(
     <Provider theme={defaultTheme}>
-      <ComboBox data-testid="test-combobox">
+      <ComboBox label="test-combobox" data-testid="test-combobox">
```

Comment:
do we need this in the example? do we need both attributes?

Reply:
I guess because of the warning about missing label? I think it's probably ok to keep the docs example minimal even so

Reply:
yeah it was cuz of the missing labels, but I'm fine getting rid of it again